src/views/ScreeningsPage.vue

```diff
@@ -39,6 +41,10 @@ const currentMoviesbyTitle = computed(() => {
         (movie) => movie.title === selectedTitle.value
       );
 });
+const showEmployeeBreadcrumbs = computed(() => {
+  return route.path === "/screenings" && userStore.isEmployee ? true : false;
```

Comment:
I think it can be simplified this way 😄 
```suggestion
  return route.path === "/screenings" && userStore.isEmployee;
```

Reply:
again much prettier 😄